pydata_sphinx_theme/docs-navbar.html

```diff
@@ -1,10 +1,21 @@
 
 <div class="container-xl">
 
+
     {% if logo %}
-    <a class="navbar-brand" href="{{ pathto(master_doc) }}">
-      <img src="{{ pathto('_static/' + logo, 1) }}" class="logo" alt="logo">
-    </a>
+      {% if not theme_logo_link %}
+      <a class="navbar-brand" href="{{ pathto(master_doc) }}">
+        <img src="{{ pathto('_static/' + logo, 1) }}" class="logo" alt="logo">
+      </a>
+      {% elif theme_logo_link[:4] == 'http' %}
+      <a class="navbar-brand" href="{{ theme_logo_link }}">
+        <img src="{{ pathto('_static/' + logo, 1) }}" class="logo" alt="logo">
+      </a>
+      {% else %}
+      <a class="navbar-brand" href="{{ pathto(theme_logo_link) }}">
+        <img src="{{ pathto('_static/' + logo, 1) }}" class="logo" alt="logo">
+      </a>      
+      {% endif %}
```

Comment:
The tests pass but I have not checked whether this really works. I keep running into a `NotImplementedError` in `asyncio` error when trying to build the HTML docs - which is apparently specific to Windows.

It'd be great if anyone can confirm if this really works :)

Reply:
Confirmed to work locally